src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/IGerritHudsonTriggerConfig.java

```diff
@@ -92,66 +92,137 @@
      */
     String getGerritCmdBuildNotBuilt();
 
+    /**
```

Comment:
Why is this change of fields (in an interface) is needed?

Reply:
Reverted the API change and marked the deprecated fields for verdict categories votes in Config to public so they can be accessed.